src/elasticodbc/es_communication.cpp

```diff
@@ -36,9 +36,9 @@ static const std::string SQL_ENDPOINT_FORMAT_JDBC =
 static const std::string PLUGIN_ENDPOINT_FORMAT_JSON =
     "/_cat/plugins?format=json";
 static const std::string OPENDISTRO_SQL_PLUGIN_NAME = "opendistro_sql";
-static const int HTTP_SUCCESS_OK = 200;
 static const std::string ALLOCATION_TAG = "AWS_SIGV4_AUTH";
 static const std::string SERVICE_NAME = "es";
+static const std::string ESODBC_PROFILE_NAME = "elasticsearchodbc";
```

Comment:
Output file of build / driver project name is elasticodbc. Do we want this to be elasticsearchodbc or just elasticodbc?

Reply:
This name will be checked against available profiles in ./aws/credentials file so it should be fine